Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -315,6 +327,6 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
   }
 
   @objc fileprivate func saveButtonTapped() {
-    self.viewModel.inputs.saveButtonTapped()
+    self.viewModel.inputs.saveButtonTapped(selected: self.saveButton.isSelected)
```

Comment:
Seeing as we need the button here, how about we change the point where we add the target to:

```swift
self.saveButton.addTarget(self, action: #selector(saveButtonTapped(_:)), for: .touchUpInside)
```

That way the button will pass itself through to the selector and we could access it like this:

```swift
@objc fileprivate func saveButtonTapped(_ button: UIButton) {
  self.viewModel.inputs.saveButtonTapped(selected: button.isSelected)
}
```